docs/README.md

```diff
@@ -100,6 +100,16 @@ go install github.com/xataio/pgroll@latest
 
 Note: requires [Go 1.21](https://golang.org/doc/install) or later.
 
+### From package manager - Homebrew
+
+To install `pgroll` with homebrew, run the following command:
+
+```sh
+# macOS or Linux
+brew tap xataio/pgroll
+brew install pgroll
+```
+
```

Comment:
Same comment in this file about reverting the other changes and leaving only these new installation instructions.

Reply:
Alright I needed to save without formatting in VSCode